ppl/cmd/zar/README.md

```diff
@@ -364,7 +384,7 @@ a count of all bytes received by 10.47.6.173 as the originator, which is the
 secondary key.  While we could build a different custom index where id.orig_h
 is the primary key, we could also just scan the custom2 index using brute force:
 ```
-zar map id.orig_h=10.47.6.173 custom2.zng | zq -f text "sum(resp_bytes)" -
+zar map id.orig_h=10.47.6.173 idx-`for file in $ZAR_ROOT/indexdefs/*; do   zq -z $file | grep 'name:"custom2.zng"' > /dev/null; if [ $? = 0 ]; then echo $file | sed 's/.*indexdefs\/idxdef-//'; fi done` | zq -f text "sum(resp_bytes)" -
```

Comment:
_This_ ugly `for` loop in backticks is a workaround for #2412. I don't know if this use case for `zar map` is something we'll want to keep supporting, but if we did address #2412, we could make this one simpler.

Reply:
```suggestion
zar map id.orig_h=10.47.6.173 idx-$(zar index ls -f zng | zq -f text 'desc="zql-custom2.zng" | cut id' -).zng | zq -f text "sum(resp_bytes)" -```